test/e2e/storage/drivers/in_tree.go

```diff
@@ -1129,6 +1129,8 @@ func (c *cinderDriver) GetDynamicProvisionStorageClass(config *storageframework.
 	parameters := map[string]string{}
 	if fsType != "" {
 		parameters["fsType"] = fsType
+	} else {
+		parameters["fsType"] = "ext4"
```

Comment:
Could you provide the reason why "ext4" is valid default value for cinder csi driver on the PR message?

/cc @oomichi 
/release-note-none